v2/contribute/decisions/session/0015-jwt-compatibility.mdx

```diff
@@ -27,5 +27,8 @@ We want to make our access token into a valid JWT. So, in the future the jwt set
 Once we make the entire access token into a JWT:
 - Add deprecation warning with a migration link and set `exposeAccessTokenToFrontendInCookieBasedAuth` to true. See [here](./0022)
 - Add a function to the FE to access the entire access token as a string.
+- Since the access token can now be used instead of the a separate JWT embedded in the access token, it is more useful to send to third-party APIs where we are not intercepting request.
+    - These will check the access token based on the JWKs endpoint exposed by the BE, but we do not need to worry about key rotation here, since the signing keys stay in the output of the jwks endpoint long after the access token expires.
+    - Calling `getAccessToken` will cause a refresh to happen if the locally stored access token is expired, so refreshing should happen automatically in most cases (if the access token doesn't expire during the request) even if we are not intercepting requests to the domain.
```

Comment:
there is an issue with this (which we may not need to solve, but worth pointing out) - if the user has email verification as required, and if they use some APi gateway for session verification (instead of using verifySession or getSession from our SDK), then it's on them to check the email verification claim in the session before granting access.

This is not communicated in the docs anywhere though, and it's not obvious that this needs to be done. I assume other auth providers solve this problem cause they don't even return the id token back to the client application if the email is not verified.